spec/support/view_component/matchers.rb

```diff
@@ -5,6 +5,12 @@ module ViewComponent
     module Matchers
       extend RSpec::Matchers::DSL
 
+      define :render do
+        match do |rendered|
+          rendered.children.any?
+        end
+      end
```

Comment:
Just because component doesn't have any children, doesn't mean it hasn't rendered, right?

Reply:
`rendered` is a nokogiri node - it has no children only when component rendered into a blank string.